calcom/cal.com
# feat: Enable Conferencing Apps for Team Events [CAL-1925]

## What does this PR do?



Please review and merge #10353 first.

This PR enables conferencing apps to be installed to a team/org and used for team event types. 

The current conferencing apps that will be enabled in this PR are:
- MS Teams
- Jitsi
- Webex
- Huddle01

Fixes #9430 



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Create an org and a team under the org
    - The org should have two members
- Create a collective, round-robin, and a managed event type
    - The collective event should only have one member assigned
- For each app listed above book the following scenarios and ensure that the meeting URLs are unique for each
    - Book a collective event
    - Book two round robin events so both members are assigned
    - Book a managed event type under both members	

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
